xml/System/String.xml

```diff
@@ -14136,10 +14197,13 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
    
   
 ## Examples  
+
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]  
+
```

Comment:
```suggestion
```

Reply:
@mairaw -- do you mean to delete blank line 14202 or the INCLUDE on line 14201?

Reply:
Usually when I leave suggestions you can just apply them directly on the GitHub UI (you can even batch them in the Files changed tab).

In this case, it was to remove the blank line.

Reply:
That's what I thought, thanks for clarifying.  The GitHub UI is not allowing me to make these changes directly, but rather is giving me an error message; refreshing the page doesn't help.